jenkins-x.yml

```diff
@@ -0,0 +1,88 @@
+pipelineConfig:
+  env:
+  - name: GIT_AUTHOR_EMAIL
+    value: jenkins-x@googlegroups.com
+  - name: GIT_AUTHOR_NAME
+    value: jenkins-x-bot
+  - name: GOPROXY
+    value: http://jenkins-x-athens-proxy
+  - name: GOOGLE_APPLICATION_CREDENTIALS
+    value: /usr/local/lib/bazel/etc/google-creds.json
+  - name: PROW_REPO_OVERRIDE
+    value: gcr.io/jenkinsxio/prow
+  pipelines:
+    pullRequest:
+      pipeline:
+        agent:
+          image: gcr.io/jenkinsxio/prow-builder:j
+        stages:
+        - name: build
+          steps:
+          - name: build
+            command: bazel
+            args: 
+            - build
+            - //prow/...
+          options:
```

Comment:
This can actually go on the `pipeline`s, rather than the `stage`s, to simplify things. =)